app/utilities/schema.py

```diff
@@ -115,13 +161,8 @@ def _load_schema_file(schema_name, language_code):
         schema_path=schema_path,
     )
 
-    try:
-        with open(schema_path, encoding="utf8") as json_data:
-            return json.load(json_data, use_decimal=True)
-
-    except FileNotFoundError as e:
-        logger.error("no schema file exists", filename=schema_path)
-        raise e
+    with open(schema_path, encoding="utf8") as json_data:
```

Comment:
The context manager isn't `json_data` so this is a sightly misleading variable name. Something like `file` or `fp` would be more accurate.